x-pack/test/apm_api_integration/common/authentication.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const APM_READ_USER = {
+  name: 'apm_read_user',
+  roles: ['apm_read_user', 'superuser'],
```

Comment:
Is there a built-in role called `apm_read_user` or is this a custom role? If it's the latter, where is the role defined?

Reply:
Dang, forgot to push my latest changes. `superuser` shouldn't be there either.